ansible/roles/master/handlers/main.yml

```diff
@@ -22,3 +22,6 @@
 
 - name: restart iptables
   service: name=iptables state=restarted
+
+- name: restart kubelet
+  service: name=kubelet state=restarted
```

Comment:
Why would there be a kubelet on the master, unless it was already a node, in which case, why would we need this?